src/Stratis.Bitcoin.IntegrationTests/Wallet/SendingTransactionOverPolicyByteLimitSpecification.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Remotion.Linq.Clauses;
+using Xunit;
+
+namespace Stratis.Bitcoin.IntegrationTests.Wallet
+{
+    public partial class SendingTransactionOverPolicyByteLimit
+    {
+        [Fact]
+        public void sending_transaction_near_policy_byte_limit()
+        {
+            Given(two_connected_nodes);
+            And(node1_builds_undersize_transaction_to_send_to_node2);
+            And(serialized_size_of_transaction_is_within_1KB_of_upper_limit);
+            When(sending_the_transaction);
+            Then(node1_succeeds_sending_tx_to_node2);
+        }
+
+        [Fact]
+        public void sending_transaction_over_policy_byte_limit()
+        {
+            Given(two_connected_nodes);
+            And(node1_builds_oversize_tx_to_send_to_node2);
+            Then(node1_fails_with_oversize_transaction_wallet_error);
```

Comment:
Should this be a wallet error? or does it mean the wallet raised the error but it failed to get in the mempool.

Reply:
Yes we get a WalletException: Could not build the transaction. Details: Transaction's size is too high. Actual value is 107344, but the maximum is 100000
I understand the Wallet gets this restriction from NBitcoin StandardTransactionPolicy::MaxTransactionSize = 100000

Reply:
Ok so the failure comes from the mempool I take it?
Hmm thats not exactly the test I wanted, I want to see a trx arrive to the mempool from a peer and fails.